### PR TITLE
Forwarding SMS to email demo

### PR DESCRIPTION
**Pull request by rsowald**

### CODE REVIEW

File: Messaging/Forward Messages to Email - Node.js/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+require("dotenv").config();
+let sw = require("@signalwire/realtime-api");
+let Messaging = sw.Messaging;
```
Comment (danieleds) [inline Messaging/Forward Messages to Email - Node.js/index.js:3]: you should be able to do something like 

```js
const { Messaging } = require("@signalwire/realtime-api")
```

so you don't need the temporary `sw` variable

File: Messaging/Forward Messages to Email - Node.js/.env.sample [hunk 2]
```diff
@@ -0,0 +1,7 @@
+SPACE_URL=<YOUR SPACE URL>
```
Comment (danieleds) [inline Messaging/Forward Messages to Email - Node.js/.env.sample:1]: it seems that this is not needed in the code? Should it be removed from here?

File: Messaging/Forward Messages to Email - Node.js/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+let mailgun = require("mailgun.js");
+
+const swClient = new Messaging.Client({
+    project: process.env.PROJECT_ID,
+    token: process.env.API_TOKEN,
+    contexts: ["office"]
+});
+
+const Mailgun = new mailgun(formData);
+const mgClient = Mailgun.client({ username: 'api', key: process.env.MAILGUN_API_TOKEN });
```
Comment (danieleds) [inline Messaging/Forward Messages to Email - Node.js/index.js:14]: the username "api" is hardcoded here; is this correct? Or should it go in the env file?

Comment (rsowald) [reply Messaging/Forward Messages to Email - Node.js/index.js:14]: https://documentation.mailgun.com/en/latest/quickstart-sending.html#send-via-api
The Mailgun documentation recommends this setup

File: Messaging/Forward Messages to Email - Node.js/.env.sample (hunk 2 above)
Comment (rsowald) [reply Messaging/Forward Messages to Email - Node.js/.env.sample:1]: you are correct. Removed

File: Messaging/Forward Messages to Email - Node.js/index.js (hunk 1 above)
Comment (rsowald) [reply Messaging/Forward Messages to Email - Node.js/index.js:3]: ok. Suggestion incorporated.

**Review by danieleds — APPROVED** [review]

**Pull request merged**